types/htmlparser2/index.d.ts

```diff
@@ -1,30 +1,20 @@
-// Type definitions for htmlparser2 v3.7.x
+// Type definitions for htmlparser2 v3.10.x
 // Project: https://github.com/fb55/htmlparser2/
 // Definitions by: James Roland Cabresos <https://github.com/staticfunction>
 //                 Linus Unnebäck <https://github.com/LinusU>
+//                 Johan Davidsson <https://github.com/johandavidson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 ///<reference types="node"/>
+///<reference types="domhandler"/>
+///<reference types="domutils"/>
 
 import { Writable } from 'stream'
+import { DomHandler } from 'domhandler';
+import * as DomUtils from 'domutils';
+export { DomElement, DomHandlerOptions, DomHandler, Element, Node } from 'domhandler';
 
-export interface Handler {
-    onopentag?: (name: string, attribs: { [type: string]: string }) => void;
-    onopentagname?: (name: string) => void;
-    onattribute?: (name: string, value: string) => void;
-    ontext?: (text: string) => void;
-    onclosetag?: (text: string) => void;
-    onprocessinginstruction?: (name: string, data: string) => void;
-    oncomment?: (data: string) => void;
-    oncommentend?: () => void;
-    oncdatastart?: () => void;
-    oncdataend?: () => void;
-    onerror?: (error: Error) => void;
-    onreset?: () => void;
-    onend?: () => void;
-}
-
-export interface Options {
+export interface ParserOptions {
```

Comment:
This will be a breaking change for anyone importing `Options`, but maybe that's okay...

We could add a `@deprecated` `export type Options = ParserOptions` though

Reply:
Yes, I guess my inexperience working with this shines through. The @ deprecated decorator doesn't work though. Would you consider `/* @deprecated */ export type Options = ParserOptions` or maybe stick with the old naming? (I guess this will be braking changes anyway since the old handler and the new DomHandler isn't the same. I just wanted to clearly separate the DomHandlerOptions from the options for the parser)